classes/notification/form/PKPNotificationsUnsubscribeForm.inc.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * @defgroup notification_form Notification Form
+ */
+
+/**
+ * @file classes/notification/form/PKPNotificationsUnsubscribeForm.inc.php
+ *
+ * Copyright (c) 2014-2019 Simon Fraser University
+ * Copyright (c) 2000-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
```

Comment:
Use GPLv3 for new code (this one is important!)

Reply:
Fixed